content/en/docs/instrumentation/net/libraries.md

```diff
@@ -1,23 +1,28 @@
 ---
-title: Automatic Instrumentation
+title: Using instrumentation libraries
 weight: 3
-linkTitle: Automatic
+linkTitle: Libraries
 ---
 
-.NET supports automatic instrumentation with [instrumentation
-libraries](/docs/reference/specification/glossary/#instrumentation-library) that
-generate telemetry data for a particular instrumented library.
+The [automatic instrumentation](/docs/reference/specification/glossary/#automatic-instrumentation)
```

Comment:
Can we drop "the" and call it "Automatic instrumentation for .NET"? I think that will be less confusing for now until we have a clearer way to explain what the mechanism is.

Reply:
I will do it tomorrow unless there will be no objections to do so.

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry.io/pull/1565/commits/739e1298b8d9df6145d4dc33992f1b2f335c0e24